viewer/src/migration.ts

```diff
@@ -2,7 +2,9 @@
  * Copyright 2020 Cognite AS
  */
 
+export { CogniteModelBase } from './migration/CogniteModelBase';
 export { Cognite3DModel } from './migration/Cognite3DModel';
+export { CognitePointCloudModel } from './migration/CognitePointCloudModel';
 export { Cognite3DViewer } from './migration/Cognite3DViewer';
 export { Intersection } from './migration/intersection';
-export { Color, Cognite3DViewerOptions, AddModelOptions } from './migration/types';
+export { Color, Cognite3DViewerOptions, AddModelOptions, WellKnownModelTypes } from './migration/types';
```

Comment:
I have made a class File3dFormat which I believe we can use instead of WellKnownModelTypes. Can also be renamed

Reply:
I'm not sure I agree with this. I saw the existing enum, but I don't think there always will be a 1-to-1 mapping between file format and model type. There are also outputs that doesn't map directly to a model that we will use in Reveal, e.g. walkable path. Therefor I think it makes sense with two enums.

Also; migration only exposes types within that package, and Files3dFormat is defined outside the package...